splunk-otel-android/src/main/java/com/splunk/rum/RumInitializer.java

```diff
@@ -116,15 +116,16 @@ SplunkRum initialize(ConnectionUtil.Factory connectionUtilFactory, Looper mainLo
                             new RumInitializer.InitializationEvent(
                                     "batchSpanProcessorInitialized", timingClock.now()));
 
-                    RumAttributeAppender attributeAppender =
-                            new RumAttributeAppender(visibleScreenTracker, connectionUtil);
+                    ScreenAttributesAppender screenAttributesAppender =
+                            new ScreenAttributesAppender(visibleScreenTracker);
                     initializationEvents.add(
                             new RumInitializer.InitializationEvent(
                                     "attributeAppenderInitialized", timingClock.now()));
 
                     tracerProviderBuilder
-                            .addSpanProcessor(attributeAppender)
+                            .addSpanProcessor(new NetworkAttributesAppender(connectionUtil))
```

Comment:
The others aren't created in-line like this, maybe just make a local var for consistency?